src/pages/resultsView/mutualExclusivity/MutualExclusivityTab.tsx

```diff
@@ -0,0 +1,183 @@
+import * as React from 'react';
+import MutualExclusivityTable from "./MutualExclusivityTable";
+import {observer} from "mobx-react";
+import {Checkbox} from 'react-bootstrap';
+import styles from "./styles.module.scss";
+import {computed, observable} from "mobx";
+import {MutualExclusivity} from "../../../shared/model/MutualExclusivity";
+import Combinatorics from 'js-combinatorics';
+import fisherTest from 'fishertest';
+
+export interface IMutualExclusivityTabProps {
+    isSampleAlteredMap: {[key: string]: boolean[]};
```

Comment:
I wonder if we can no longer consider sampleIds unique since we're handling multiple studies?  : (